resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -0,0 +1,242 @@
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    DirField,
+    FileHtmlField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField
+)
+
+
+def create_symlink(src, dst):
+    """Create a symbolic link."""
+    return Cmd['ln']('-s', '--backup=numbered', src, dst)
+
+
+def create_summary_table(samples, species, build):
+    """Prepare sample summary MultiQC table."""
+    sample_summary = {
+        "id": "sample_summary",
+        "section_name": "Sample Summary",
+        "plot_type": "table",
+        "file_format": "json",
+        "data": {}
+    }
+
+    for sample in zip(samples, species, build):
+        sample_name, species, build = sample
+        if build not in ['rRNA', 'globin']:
+            sample_summary['data'][sample_name] = {
+                'Species': species,
+                'Genome Build': build,
+            }
+
+    with open('sample_data_mqc.json', 'w') as out_file:
+        json.dump(sample_summary, out_file)
+
+
+class Multiqc(Process):
+    """Aggregate results from bioinformatics analyses across many samples into a single report.
+
+    [MultiQC](http://www.multiqc.info) searches a given directory for analysis logs and compiles a
+    HTML report. It's a general use tool, perfect for summarising the output from numerous
+    bioinformatics tools.
+    """
+
+    slug = 'multiqc'
+    process_type = 'data:multiqc'
+    name = 'MultiQC'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image': 'resolwebio/common:1.3.1'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 8192,
+        },
+    }
+    entity = {
+        'type': 'sample',
+    }
+    category = 'Other'
+    data_name = 'MultiQC report'
+    version = '1.4.0'
+
+    class Input:
+        """Input fields to process ReverseComplementSingle."""
+
+        data = ListField(
+            DataField(
+                data_type='',
+                description='Select multiple data objects for which the MultiQC report is to be '
+                            'generated.'
+            ),
+            label='Input data'
+        )
+
+        class Advanced:
+            """Options."""
+
+            dirs = BooleanField(
+                label='--dirs',
+                default=True,
+                description='Prepend directory to sample names.',
+            )
+
+            dirs_depth = IntegerField(
+                label='--dirs-depth',
+                default=-1,
+                description='Prepend a specified number of directories to sample names. Enter a '
+                            'negative number to take from start of path.',
+            )
+
+            fullnames = BooleanField(
+                label='--fullnames',
+                default=False,
+                description='Do not clean the sample names (leave as full file name).',
```

Comment:
I would write what this flag does (and that it's default is False) and not what it doesn't do.